internal/fetch/lastSuccessfulMeta_test.go

```diff
@@ -236,8 +26,28 @@ func (m *MockHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	m.Called(w, r)
 }
 
-func TestLastSuccessfulMetaRequest_LastSuccessfulMetaURL(t *testing.T) {
-	for _, tc := range []struct {
+type LastSuccessfulMetaSuite struct {
+	suite.Suite
+	JobsJson               string
```

Comment:
could you change it to `JobsJSON` to stick to the Go's (opinionated) convention of abbreviations / "initialisms"?  https://github.com/golang/go/wiki/CodeReviewComments#initialisms  (can do in a follow-up)